src/item.cpp

```diff
@@ -6462,21 +6462,9 @@ bool item::units_sufficient( const Character &ch, int qty ) const
     return units_remaining( ch, qty ) == qty;
 }
 
-item::reload_option::reload_option( const reload_option &rhs ) :
-    who( rhs.who ), target( rhs.target ), ammo( rhs.ammo ),
-    qty_( rhs.qty_ ), max_qty( rhs.max_qty ), parent( rhs.parent ) {}
+item::reload_option::reload_option( const reload_option &rhs )  = default;
```

Comment:
Can't you see it? 😃 
```suggestion
item::reload_option::reload_option( const reload_option &rhs ) = default;
```

Reply:
Thanks.  `clang-tidy` made the change, not me :).